src/network_manager.cpp

```diff
@@ -522,5 +529,44 @@ void Manager::handleAdminState(std::string_view state, unsigned ifidx)
     }
 }
 
+void Manager::writeLLDPDConfigurationFile()
+{
+    std::ofstream lldpdConfig(lldpFilePath);
+
+    for (const auto& intf : interfaces)
+    {
+        bool emitlldp = intf.second->EthernetInterfaceIntf::emitLLDP();
```

Comment:
intf.second->emitLLDP(); this should have worked right?
This gives the value of emitLLDP of that particular interface I believe.

Reply:
yes, it works

Reply:
done